README.md

```diff
@@ -36,7 +36,7 @@ Pre for `<pre>` block, code for `<code>` span snippet.
 
 Clone the repo:
 
-    $ git clone git@github.com:haskell-infra/hl.git
+    $ git clone https://github.com/haskell-infra/hl
```

Comment:
Perhaps split it into two here: "For anonymously cloning the repository, use ..., otherwise use ...."

Though it probably doesn't matter much. If someone is reading that readme, they most likely know how to clone the repo from GitHub already.